tsdb/series_set.go

```diff
@@ -126,6 +135,15 @@ func (s *SeriesIDSet) AndNot(other *SeriesIDSet) *SeriesIDSet {
 	return &SeriesIDSet{bitmap: roaring.AndNot(s.bitmap, other.bitmap)}
 }
 
+// Or returns a new SeriesIDSet containing elements that were present in s or other.
+func (s *SeriesIDSet) Or(other *SeriesIDSet) *SeriesIDSet {
```

Comment:
It seems like `a.Or(b)` is equivalent to:

```go
ss := NewSeriesIDSet()
ss.Merge(a, b)
```

Since we already have a `Merge` method available. Maybe this is faster than `Merge` as we don't need to allocate a `[]*roaring.Bitmap`?

Reply:
Fixed in 3b77f18.